### PR TITLE
useParams() to print a parameter in the url to the console

### PR DESCRIPTION
**Pull request by schaferyan**
Currently if you visit http://localhost:3000/play/[params] where params is any string of text, the gameboard is displayed as usual but the value of params is logged to the console.

### CODE REVIEW

File: src/Components/GameSession.js [hunk 1]
```diff
@@ -6,17 +6,19 @@ import wordImporter from '../funcLib/WordImporter.js';
 import PlayAgainButton from './PlayAgainButton.js';
 import { Col, Container, Row } from 'react-bootstrap';
 import checkForBingo from '../funcLib/CheckForBingo';
-import { useOutletContext } from 'react-router-dom';
+import { useOutletContext, useParams } from 'react-router-dom';
```
Comment (nojronatron) [inline src/Components/GameSession.js:9]: If multiple params, does the code still work as expected?

Comment (schaferyan) [reply src/Components/GameSession.js:9]: Yes!

Comment (schaferyan) [reply src/Components/GameSession.js:9]: Multiple params are included like: 

[localhost:3000/play/[params1]/[params2] ](localhost:3000/play/params1/params2)

then referenced in order when useParams() is called

**Review by nojronatron — APPROVED** [review]
Very good. Keep the multi-param thing in mind. Not sure that will be necessary but it would be good to handle that outlier case properly. Go ahead and merge into main if there are no other changes needed.

**Pull request merged**